lib/engine/config/game/g_1846.rb

```diff
@@ -399,6 +399,7 @@ module G1846
       "abilities": [
         {
           "type": "token",
+          "name": "Reserved $40/$60 Ft. Wayne token",
```

Comment:
i think we shouldn't use name, it should be description

can you change the ability.name to be ability.description?

Reply:
ok, changed

Reply:
Ok, fixed